### ISSUE TITLE
Rooms list is less keyboard-accessible

### ISSUE THREAD

**Issue by ndarilek** [vector-im/element-web#21436]
### Steps to reproduce

1. From a room, press ctrl-k.
2. Press down arrow.
3. Press tab a few times.

#### What did you expect?

I could actually *reach* the rooms tree via the keyboard like I used to, and navigate it as well.

#### What happened instead?

The room tree cannot be focused unless I use caret mode to hunt up the tree. At that point it becomes navigable.

Also, this seems somewhat inconsistent. Sometimes it works, sometimes it doesn't. It seems more prone to failure if I do it from a room vs. from the home screen reached via ctrl-alt-h. What *doesn't* work and used to is using down arrow to quickly travel from the search field to the list of rooms. I used that as a shortcut to reach rooms--ctrl-k, down arrow--but now that never accessibly moves focus to the rooms tree.

### Operating system

Windows/NVDA

### Application version

1.10.7

### How did you install the app?

element.io

### Homeserver

Synapse 1.54

### Will you send logs?

No

**Comment by albertotirla** [comment]
yes, I can confirm this happening as well, I'm on ubuntu 20.04, so it's manifesting across platforms.

**Comment by vojtapolasek** [comment]
I can confirm this on Arch Linux, element-desktop 1.10.7-1.
It seems that I can't overcome this issue by any tabbing.

**Comment by pvagner** [comment]
Hello,
My workaround is as follows:
* Press ctrl+k to jump to the filter entry
* Type in a few characters
* Press tab key several times to jump to the room list.
Unfortunatelly it's not very pleasant experience for screen reader users.

**Comment by t3chguy** [comment]
Regressed by 6e143c313ed139b5599f252dc53350886363b6c9
